code/Items/Data/ItemClothesData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using ssl.Items.Carriables;
+using ssl.Player;
+
+namespace ssl.Items.Data
+{
+    public class ItemClothesData : ItemData
+    {
+        public ItemClothesData(string id, string name, string model, ClothesSlot slot) : base(id, name, model)
+        {
+            Slot = slot;
+        }
+
+        public ClothesSlot Slot { get; }
+        
+        public override Item create()
```

Comment:
lowercase is for java.
Do you mean Create ?

Reply:
Yep